Safety/emergency-response-guide.rst

```diff
@@ -0,0 +1,173 @@
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Giovanni Corvetto*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Safety-emergency-response-guide:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+########################
+Emergency Response Guide
+########################
+
+This is a basic emergency response guide organized by the type of emergency. 
+This is not a comprehensive first responder training guide.
+This does not replace safety training required to be on the summit.
+It is recommended to contact the safety team to complete summit driver's training in addition to the required summit training. 
+
+.. note::
+    Observing specialists serve as the Nighttime Safety Officers if there is not a member of the safety team physically on the summit.
+    If they take the decision to close the telescope or evacuate the summit for inclement weather, that decision is not negotiable.
+    The priority of safety will always be given to staff first, equipment second. 
+
+.. _emergency-response-guide-medical-emergency:
+
+Medical Emergency
+^^^^^^^^^^^^^^^^^
+
+.. Preconditions
+
+Contact the PolyClinic Paramedic at extension 577
```

Comment:
Just for my personal knowledge, I've never heard it referred to as "Polyclinic" -- what is that?
Is it the company that provides the service or something?

Reply:
Yes, it is the name on the door and for it is referred to on the summit. I clarified that they were the paramedic on summit.